### PR TITLE
pfsshell: init at 1.1.1

### PR DESCRIPTION
**Pull request by makefu**
###### Motivation for this change

init pfsshell, which is being used in the tutorial for https://nixos.wiki/wiki/Playstation2
#79142

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/pfsshell/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  # Build errors since 1.1.1 when format hardening is enabled:
+  #   cc1: error: '-Wformat-security' ignored without '-Wformat' [-Werror=format-security]
+  hardeningDisable = [ "format" ];
+
+  meta = with stdenv.lib; {
+    inherit (src.meta) homepage;
+    description = "PFS (PlayStation File System) shell for POSIX-based systems";
+    platforms = platforms.unix;
+    license = licenses.gpl2Only; #The APA, PFS, and iomanX libraries are licensed under the The Academic Free License version 2.
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/pfsshell/default.nix:24]: Would you be so kind and add those licenses to here and maybe nixpkgs?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101248` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pfsshell</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101248` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pfsshell</li>
  </ul>
</details>

**Comment by makefu** [comment]
@SuperSandro2000 i added afl20 to licenses

File: pkgs/tools/misc/pfsshell/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  #   cc1: error: '-Wformat-security' ignored without '-Wformat' [-Werror=format-security]
+  hardeningDisable = [ "format" ];
+
+  meta = with stdenv.lib; {
+    inherit (src.meta) homepage;
+    description = "PFS (PlayStation File System) shell for POSIX-based systems";
+    platforms = platforms.unix;
+    license = with licenses; [
+      gpl2Only # the pfsshell software itself
+      aflv20    # APA, PFS, and iomanX libraries which are compiled together with this package
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/pfsshell/default.nix:26]: Either with our without v otherwise it wont work.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/pfsshell/default.nix (hunk 2 above)
Comment (makefu) [reply pkgs/tools/misc/pfsshell/default.nix:26]: true haha, let me fix it

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101248` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pfsshell</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 101248` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pfsshell</li>
  </ul>
</details>

**Pull request merged**